packages/devtools_app/lib/src/performance/performance_controller.dart

```diff
@@ -208,15 +210,17 @@ class PerformanceController extends DisposableController
       await processTraceEvents(allTraceEvents);
       _processing.value = false;
 
+      _timelinePollingRateLimiter = RateLimiter(
+        timelinePollingRateLimit,
+        () async => await _pullTraceEventsFromVmTimeline(),
```

Comment:
replace:
`() async => await _pullTraceEventsFromVmTimeline()`
with
`_pullTraceEventFromVmTimeline`

@pq is there a lint tracking this case?

Reply:
done

Reply:
Hrmmm, closest thing would be `unnecessary_lambdas`.